zips/zip-0231.md

```diff
@@ -213,7 +230,7 @@ wrong counter value), ensuring that a malformed memo is not returned.
 | Bytes    | Name          | Data Type                      | Description                                                  |
 |----------|---------------|--------------------------------|--------------------------------------------------------------|
 |   1      | `fAllPruned`  | `uint8`                        | 1 if all chunks have been pruned, otherwise 0.               |
-|   32     | `nonceOrHash` | `byte[32]`                     | The nonce for deriving encryption keys, or the overall hash. |
+|   32     | `saltOrHash`  | `byte[32]`                     | The salt for deriving encryption keys, or the overall hash.  |
```

Comment:
I think nonce here gets confused with the nonce we create for each chunk encryption.